identity-issuance/src/main/java/com/android/identity/issuance/evidence/EvidenceRequestEId.kt

```diff
@@ -0,0 +1,8 @@
+package com.android.identity.issuance.evidence
+
+/**
+ * Evidence request type for asking user to identify themselves using German eID card.
+ */
+data class EvidenceRequestEId(
```

Comment:
I think we should probably refer to the German EID card as such and not just "EID"? This is b/c I imagine we'd add support for other EID cards in the future too, many many countries have them already (for example Japan has MNC and JPKI). Proposal

- in prose (e.g. commit message, comments, etc) use "German eID card, according to BSI TR-03127"
- for types use `GermanEidCard`, e.g. `EvidenceRequestGermanEidCard`
- for variables and strings use `germanEidCard`

Reply:
Done

Reply:
done